src/validation.h

```diff
@@ -469,13 +469,13 @@ bool ResetBlockFailureFlags(CBlockIndex* pindex) EXCLUSIVE_LOCKS_REQUIRED(cs_mai
 extern CChain& chainActive;
 
 /** Global variable that points to the coins database (protected by cs_main) */
-extern std::unique_ptr<CCoinsViewDB> pcoinsdbview;
+extern std::unique_ptr<CCoinsViewDB> pcoinsdbview GUARDED_BY(cs_main);
 
 /** Global variable that points to the active CCoinsView (protected by cs_main) */
-extern std::unique_ptr<CCoinsViewCache> pcoinsTip;
+extern std::unique_ptr<CCoinsViewCache> pcoinsTip GUARDED_BY(cs_main);
 
 /** Global variable that points to the active block tree (protected by cs_main) */
-extern std::unique_ptr<CBlockTreeDB> pblocktree;
+extern std::unique_ptr<CBlockTreeDB> pblocktree GUARDED_BY(cs_main);
 
 /**
  * Return the spend height, which is one more than the inputs.GetBestBlock().
```

Comment:
Missing annotation for this function

Reply:
@MarcoFalke Do you mean `GetSpendHeight`? `GetSpendHeight` locks `cs_main` itself :-)